dev/system-test/firestore.ts

```diff
@@ -2540,24 +2571,28 @@ describe('Transaction class', () => {
     expect(snapshot.get('foo')).to.equal(1);
   });
 
-  it('fails read-only with writes', async () => {
-    let attempts = 0;
+  // This is not supported when working with the emulator. Do not run tests against emulator.
```

Comment:
Please define "This".

Reply:
Updated for clarity and consistency. PTAL.